olmeiro-orozco/tarea-2/tarea2.1-Cover/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Cover Magazine</title>
+  </head>
+
+  <body>
+    <main>
+      <header>
+        <h1>business</h1>
+      </header>
+
+      <section class="date-web">
+        <p class="first-paragraph">january 2022/vol. 001</p>
+        <p class="second-paragraph">www.businessmagazin.com</p>
+      </section>
+
+      <section class="coverphoto">
+        <section class="coverphoto__photo">
+          <figure>
+            <img
+              src="./assets/personalPhoto.png"
+              alt="Olmeiro personal photo"
+            />
+          </figure>
+          <section class="coverphoto__photo-text">
+            <p>
+              work hard <br/>
+              <span>get success</span>
+            </p>
+          </section>
+          <section class="centrado"></section>
```

Comment:
@OlmeiroOrozco this should be a psuedo element like ::after

Reply:
I will do it.